tsconfigdoc.json

```diff
@@ -230,6 +233,9 @@
     {
       "path": "./packages/statusbar-extension"
     },
+    {
+      "path": "./packages/tabmanager-extension"
```

Comment:
Ah it looks like these entries are back.

Wondering if removing the directory on disk would fix it.

Reply:
Just did and I think it fixed it :D Thanks!